tests/backend/user/test_user_create.py

```diff
@@ -201,10 +210,8 @@ async def test_user_create_invalid_certificate(alice_ws, alice, bob, mallory):
             redacted_user_certificate=good_user_certificate,
             redacted_device_certificate=good_device_certificate,
         )
-        assert rep == {
-            "status": "invalid_certification",
-            "reason": "Invalid certification data (Signature was forged or corrupt).",
-        }
+        # The reason is no longer generated
```

Comment:
no need for this comment I think: it is interesting in this PR to explain why the reason is no longer verified, but when reading the code alone, it seems totally fine without a check to the reason field

Reply:
Ok !